build_tools/azure/install_win.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
-
-set -e
-set -x
-
-# defines the get_dep and show_installed_libraries functions
-source build_tools/shared.sh
-
-if [[ "$DISTRIB" == "conda" ]]; then
-    # Install/update conda with the libmamba solver because the legacy solver
-    # can be slow at installing a specific version of conda-lock.
-    conda install -n base conda conda-libmamba-solver -y
-    conda config --set solver libmamba
-    conda install -c conda-forge "$(get_dep conda-lock min)" -y
-    conda-lock install --name $VIRTUALENV $LOCK_FILE
-    source activate $VIRTUALENV
-else
-    python -m venv $VIRTUALENV
-    source $VIRTUALENV/Scripts/activate
-    pip install -r $LOCK_FILE
-fi
-
-show_installed_libraries
-
-# Build scikit-learn
-python setup.py bdist_wheel
-
-# Install the generated wheel package to test it
-pip install --pre --no-index --find-links dist scikit-learn
```

Comment:
Using `install.sh` now results in installing sklearn via ``python setup.py develop`` but I guess it's fine. Would it be possible to enable meson instead ?

Reply:
I think this is fine too. I am not sure why Windows was different to be honest, maybe an historical thing where we were using a batch file before switching for bash?

It will be possible to use Meson if we want. Some testing in #28040 indicated that it was working fine on Windows.

Eventually, I have some plans to switch to Meson for all builds but one to keep testing building with setuptools.

Reply:
> maybe an historical thing where we were using a batch file before switching to bash?

Yup, this is why there is a separate file in windows.